app/services/board/searcher.rb

```diff
@@ -10,7 +10,10 @@ def search(params)
   end
 
   def search_authors(author_ids)
-    author_boards = author_ids.map { |author_id| BoardAuthor.where(user_id: author_id).pluck(:board_id) }.reduce(:&).uniq
+    # get author matches for boards that have at least one
+    author_boards = BoardAuthor.where(user_id: author_ids, cameo: false).group(:board_id)
```

Comment:
(We don't want to include cameos? Huh, I kinda figured that might be something people wanted, but if that was the original issue, then it seems OK)

Reply:
I'd be fine with writing a way *to* include cameos but it was a source of confusion especially since the cameo authors weren't displayed in results